code/src/index.css

```diff
@@ -1,13 +1,197 @@
   -moz-osx-font-smoothing: grayscale;
+  background-image: url(../public/bree-anne-XnpWqPZNfXE-unsplash.jpg);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100vw;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.paper {
+  width: 98vw;
+  height: auto;
+  min-height: 80vh;
+  background-color: #fff;
+  box-shadow: 3px 2px 2px 0px rgba(0,0,0,0.4);
+  position: relative;
+  padding: 40px 0 40px 0;
+  margin: 20px 0 10px 0;
+}
+.paper::before {
+  content: '';
+  width: 2px;
+  height: 100%;
+  background-color: rgba(255, 0, 0, 0.6);
+  position: absolute;
+  top: 0;
+  left: 40px;
+}
+.pattern {
+  min-height: 80vh;
+  height: auto;
+  background-image: repeating-linear-gradient(white 0px, white 24px, teal 25px);
+}
+.content {
+  padding-top: 6px;
+  padding-left: 56px;
+  padding-right: 16px;
+  line-height: 25px;
+  font-family: 'Shadows Into Light', cursive;
+  font-size: 1rem;
+  letter-spacing: 1px;
+  word-spacing: -5px;
+}
+.header {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 1.2rem;
+}
+h1 {
+  font-size: 1.2rem;
+  line-height: 1.2rem;
+}
+.total {
+  text-align: right;
+}
+.remove-btn {
+  border: none;
+  color: rgba(255, 0, 0, 0.8);
+  background-color: transparent;
+  font-size: 1rem;
+  cursor: pointer;
+  margin-left: 0.8rem;
+  margin-top: -4px;
+  position: absolute;
+  right: 2vw;
+  text-shadow: 1px 1px 2px rgba(0,0,0,0.2);
+}
+li p {
+  display: flex;
+  flex-direction: row;
+}
+.container {
+  display: block;
+  position: absolute;
+  margin: 2px 0 0 10px;
+  cursor: pointer;
+  font-size: 22px;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+  right: 15vw;
+}
+.container input {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  height: 0;
+  width: 0;
+}
+.checkmark {
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 0.7rem;
+  width: 0.7rem;
+  border: 1px solid rgba(0,0,0,0.2);
+  border-radius: 2px;
+  box-shadow: 1px 1px 2px rgba(0,0,0,0.2);
+}
+.checkmark:after {
+  content: "✓";
+  position: absolute;
+  display: none;
+  font-size: 1.4rem;
+  color: rgb(0, 170, 0);
+  font-family: monospace;
+  font-style: normal;
+  line-height: 0;
+  margin-top: 4px;
+  margin-left: 1px;
+}
+.container input:checked ~ .checkmark:after {
+  display: block;
+}
+.container .checkmark:after {
+  -webkit-transform: rotate(348deg);
+  -ms-transform: rotate(348deg);
+  transform: rotate(5deg);
+} 
+.deleteall-btn img{
+  width: 150px;
+}
+.deleteall-btn {
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  width: 150px;
+  rotate: 15deg;
+  position: relative;
+  top: 4vh;
+  left: 20%;
+}
+input {
+  width: 150px;
+  border: none;
+  background-color: transparent;
+  margin-right: 10px;
+  font-family: 'Shadows Into Light', cursive;
+  outline: rgba(0,0,0,0.2);
+  font-size: 1rem;
+  line-height: 0;
+}
```

Comment:
Really like how the input field is hidden, even though it's hard to find once used.